recipes/awacs/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "awacs" %}
+{% set version = "0.9.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/cloudtools/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "9baaf8db8c4e384ac57e3d674130696b6ed2a54a70f8462a827d43da875d0ea8"
+
+build:
+  number: 0
```

Comment:
Can this be `noarch: python`?

Reply:
It uses 2to3, so I don't think so

Reply:
Can you add a TODO here to make it noarch when python2 is dropped?

Reply:
Done